app/main.py

```diff
@@ -1 +1,26 @@
-# write your code here
+from app.cafe import Cafe
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except NotVaccinatedError:
+            return "All friends should be vaccinated"
+        except OutdatedVaccineError:
+            return "All friends should be vaccinated"

```

Comment:
You can join these 2 handlers to one, because `NotVaccinatedError` and `OutdatedVaccineError` have the same parent
```suggestion
        except VaccineError:
            return "All friends should be vaccinated"
```